technique/muon/caen_thresholds.py

```diff
@@ -0,0 +1,220 @@
+import csv
+import os
+
+import genie as g
+import genie_advanced
+
+
+class CaenThresholdSetter:
+    def __init__(self,
+                 threshold_scale_default: float,
+                 absolute_threshold_default: int,
+                 dsc_width_default: float) -> None:
+        # default value to scale all thresholds by
+        self.threshold_scale_default = threshold_scale_default
+        # default value for thresholds
+        self.absolute_threshold_default = absolute_threshold_default
+        # default value to write to discriminator output width register
+        self.dsc_width_default = dsc_width_default
+        self.enable_table_path = os.path.join(
+            "C:\\Instrument\\Settings\\config\\NDXCHRONUS\\Python\\tables",
```

Comment:
The version on ARGUS has a different path here which looks at NDXARGUS, this path probably needs to determine which instrument it is on, rather than this being a magic value.